Kernel/Bus/VirtIO/RNG.h

```diff
@@ -26,7 +26,7 @@ class RNG final
     virtual void initialize() override;
 
 private:
-    virtual StringView class_name() const override { return "VirtIOConsole"; }
+    virtual StringView class_name() const override { return "VirtIOConsole"sv; }
```

Comment:
This looks like a copy-paste-error from the past. Shouldn't this be `return "VirtIO::RNG"sv;` or something?

Reply:
Yup, fixed. Thanks!